drivers/hmis/app/models/hmis/form/form_processor.rb

```diff
@@ -30,6 +30,8 @@ class Hmis::Form::FormProcessor < ::GrdaWarehouseBase
   belongs_to :youth_education_status, class_name: 'Hmis::Hud::YouthEducationStatus', optional: true, autosave: true
   belongs_to :employment_education, class_name: 'Hmis::Hud::EmploymentEducation', optional: true, autosave: true
   belongs_to :current_living_situation, class_name: 'Hmis::Hud::CurrentLivingSituation', optional: true, autosave: true
+  belongs_to :ce_assessment, class_name: 'Hmis::Hud::Assessment', optional: true, autosave: true
+  belongs_to :ce_event, class_name: 'Hmis::Hud::Event', optional: true, autosave: true
```

Comment:
This relationship doesn't exist in the database, so we should remove this.
We would only need this if `Hmis::Hud::CustomAssessment` are generating Events/Assessments, which they're not (yet). 

For "record-editing forms" that use the SubmitForm mutation, the FormProcessor record isn't actually persisted.